packages/api/core/src/util/publish-state.ts

```diff
@@ -59,10 +59,6 @@ export default class PublishState {
     return crypto.createHash('SHA256').update(content).digest('hex');
   }
 
-  setState(state: ForgeMakeResult) {
-    this.state = state;
-  }
```

Comment:
I didn't really see a point in having a setter like this if we're going to also need to retrieve the value:

https://github.com/electron-userland/electron-forge/blob/f93da7a58bef25723f9495cb07c1080bffbb224b/packages/api/core/src/api/publish.ts#L95

If the getter/setter needs to be more complicated in the future, it seems like `get state()` and `set state(value)` would work.